docs/components/gantry/single-axis.md

```diff
@@ -57,8 +57,50 @@ The following attributes are available for `single-axis` gantries:
+| `limit_pin_enabled_high` | boolean | Optional | Whether the limit pin must be “high” or “low” to be considered “hit”, with a value of `true` representing "high". This attribute is **required** if `limit_pins` is set.<br> Default: `false` |
 | `mm_per_rev` | int | **Required** | How far the gantry moves (linear, distance in mm) per one revolution of the motor’s output shaft. This typically corresponds to Distance = PulleyDiameter * pi, or the pitch of a linear screw. |
 | `gantry_mm_per_sec` | int | Optional | The speed at which the gantry moves in millimeters per second. Used to calculate the gantry `motor`'s revolutions per minute (RPM). <br> Default: `100` RPM |
+
+## Specify the order of homing routines
+
+When `limit_pins` is set, a `single-axis` gantry component performs a homing routine when `viam-server` starts.
+The axis moves towards one limit switch, then the other, before stopping at 80% of the travel of the axis.
+
+If you configure multiple `single-axis` gantries (for example, when using a [`multi-axis` gantry](/components/gantry/multi-axis/)), the order in which these homing routines are executed may be important.
+For example, if there is a tool mounted to the z axis gantry that could be damaged during homing of the x or y axes gantries, you would want to ensure that the z axis gantry completes its homing routine first, so that the tool head returns to a safe location before the homing process begins on the other axes.
+
+In this case, you can specify the axes that must be homed first in a `depends_on` array to each `single-axis` gantry.
+The example below configures `myFirstGantry` to wait until the homing process for `mySecondGantry` has completed before starting its own:
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "components": [
+        // < Your motor config >
+        {
+            "name": "myFirstGantry",
+            "type": "gantry",
+            "model": "single-axis",
+            "attributes": {
+                "length_mm": 1000,
+                "board": "local",
+                "limit_pin_enabled_high": false,
+                "limit_pins": [
+                    "32",
+                    "36"
+                ],
+                "depends_on": [
+                    "mySecondGantry"
+                ]
+            }
+        },
+        // < mySecondGantry config >
+    ]
+}
+```
+
+{{% alert="Note" color="note" %}}
+The order of axes in the `subaxes_list` for a [`multi-axis` gantry](/components/gantry/multi-axis/) does not influence the order of homing.
```

Comment:
now the order is the order they will home in if `Home` is called on the multi-axis gantry